pinax/blog/views.py

```diff
@@ -55,7 +55,8 @@ def get_queryset(self):
 class SectionIndexView(BlogIndexView):
 
     def get_current_section(self):
-        return Section.objects.get(slug__iexact=self.kwargs.get("section"))
+        section = get_object_or_404(Section, slug__iexact=self.kwargs.get("section"))
```

Comment:
Better to generate a 404 than DoesNotExist exception if someone uses an unknown section slug.